datasets/il/mod_terrorists/crawler.py

```diff
@@ -9,28 +9,27 @@
 
 ORG_URL = "https://nbctf.mod.gov.il/he/Announcements/Documents/NBCTFIsrael%20-%20Terror%20Organization%20Designation%20List_XL.xlsx"
 PEOPLE_URL = "https://nbctf.mod.gov.il/he/Announcements/Documents/NBCTF%20Israel%20designation%20Individuals_XL.xlsx"
-FORMATS = ["%d/%m/%Y", "%d.%m.%Y", "%Y-%m-%d"]
 NA_VALUE = re.compile(r"^[\-\/]+$")
 END_TAG = re.compile(r"בוטל ביום", re.U)
 SPLITS = ["; ", "Id Number", "a) ", "b) ", "c) ", " :", "\n"]
 
 
-def parse_date(date):
+def parse_date(context: Context, date):
     dates = []
-    for part in h.multi_split(date, ["OR", ";", " - "]):
-        dates.extend(h.parse_date(part, FORMATS))
+    for part in h.multi_split(date, ["OR", ";", " - ", "a) ", "b) ", "c) "]):
+        dates.extend(h.extract_date(context.dataset, part))
     return dates
```

Comment:
do we need this to extract? are we using the dates anywhere in a way they need to be parsed, or can we just split and apply in this crawler?